### PR TITLE
Blog template setup for netlifyCMS page creation

### PR DESCRIPTION
**Pull request by slimekat**
## Changes

Ported over structure from netlify's own gatsby CMS starter,
Restructured components in preparation of a graphQL createPage API based system
## Out of Scope / Future Work

Port over template based system into current Post.js

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        },
+      })
+    })
+
+  })
+}
+
+exports.onCreateNode = ({ node, actions, getNode }) => {
+  const { createNodeField } = actions
+//   fmImagesToRelative(node)  convert image paths for gatsby images
```
Comment (lynncyrin) [inline gatsby-node.js]: _continuing https://github.com/lynncyrin/lynncyrin-dot-me/pull/242#discussion_r336289392_

⭐⭐⭐ _(must change)_ Yes please add a comment about this!

File: src/components/Layout.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        />
+        <meta name="theme-color" content="#fff" />
+
+        <meta property="og:type" content="business.business" />
+        <meta property="og:title" content={title} />
+        <meta property="og:url" content="/" />
+        <meta
+          property="og:image"
+          content={`${withPrefix('/')}img/og-image.jpg`}
+        /> */}
```
Comment (lynncyrin) [inline src/components/Layout.js]: _(non-blocking question)_ this is commented out b/c the image stuff is still WIP, yeah?

Comment (slimekat) [reply src/components/Layout.js]: Yep, I'll comment about that

File: gatsby-node.js (hunk 1 above)
Comment (slimekat) [reply gatsby-node.js]: Will do

**Pull request merged**